Formula/libsquish.rb

```diff
@@ -0,0 +1,26 @@
+class Libsquish < Formula
+  desc "Library for compressing images with the DXT standard."
+  homepage "https://sourceforge.net/projects/libsquish/"
+  url "https://downloads.sourceforge.net/project/libsquish/libsquish-1.14.tgz"
+  sha256 "5ea955dc7b566d8c30b321e09d35ad7dc7c2dfa0c3330829b034f69cf92ebc7f"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cc").write <<-EOF.undent
```

Comment:
Minor nit, but we use `EOS.undent` and `EOS` mostly elsewhere so would be nice to be consistent here.